### ISSUE TITLE
Calendar icon not visible in high contrast mode

### ISSUE THREAD

**Issue by MaheshakaMudli** [microsoft/fluentui#14234]
#### Actual behavior:

calendar icon is not at all visible in high contrast theme and very low visibility in dark theme.

#### Expected behavior:

calendar icon should be visible in all themes.

### Priorities and help requested:

Are you willing to submit a PR to fix?No

Requested priority: High

Products/sites affected: (if applicable) https://fluentsite.z22.web.core.windows.net/components/input/definition

**Comment by jurokapsiar** [comment]
Hi @MaheshakaMudli, thanks for reporting.

Can you please clarify the repro steps? You mention input in. the affected sites, but that does not use Calendar icon.
Please modify following codesandbox to show what does not work: https://codesandbox.io/s/19bsk?module=/example.js
